modules/core/include/opencv2/core/fast_math.hpp

```diff
@@ -201,7 +201,7 @@ cvRound( double value )
 {
 #if defined CV_INLINE_ROUND_DBL
     CV_INLINE_ROUND_DBL(value);
-#elif ((defined _MSC_VER && defined _M_X64) || (defined __GNUC__ && defined __SSE2__)) && !defined(__CUDACC__)
+#elif ((defined _MSC_VER && defined _M_X64) || (defined __GNUC__ && defined __x86_64__ && defined __SSE2__)) && !defined(__CUDACC__)
```

Comment:
Condition of include should be changed instead.

Reply:
I actually was wondering whether it made sense to check for __x86_64__ AND __SSE2__.
Done